projects/opencv/readnetfromtensorflow_fuzzer.cc

```diff
@@ -0,0 +1,75 @@
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/*
+ * This fuzzer is generated by UTopia project based on TEST(Test_Tensorflow, read_inception).
+ * (UTopia Project: https://github.com/Samsung/UTopia)
+ */
+#include <opencv2/dnn/dnn.hpp>
+#include <opencv2/imgcodecs.hpp>
+#include <opencv2/imgproc.hpp>
+#include <fuzzer/FuzzedDataProvider.h>
+#include <fstream>
+
+using namespace cv;
+using namespace dnn;
+
+bool saveFile(std::string Path, std::string Content) {
+  std::ofstream OFS(Path);
+  if (!OFS.is_open())
+    return false;
+
+  OFS << Content;
+  return true;
+}
```

Comment:
I think it'd be better to do all of this in-memory and not use temporary files. So, take a look at [this ](https://docs.opencv.org/4.x/d6/d0f/group__dnn.html#gacdba30a7c20db2788efbf5bb16a7884d) overload.

Reply:
4d967c2: Thanks, I use buffer to invoke readNetFromTensorflow instead of filepath.